optimum/intel/openvino/modeling_decoder.py

```diff
@@ -44,6 +45,23 @@
 
 core = Core()
 
+@dataclass
+class OVCausalLMOutputWithPast(ModelOutput):
+    """
+    Base class for causal language model (or autoregressive) outputs.
+
+    Args:
+        infer_request(`openvino.runtime.InferRequest` to be reused in the generation cycles.
+        beam_idx (`torch.Tensor` beam search algorimth context for the generation using stateful models
+    """
+
+    loss: Optional[torch.FloatTensor] = None
+    logits: torch.FloatTensor = None
+    past_key_values: Optional[Tuple[Tuple[torch.FloatTensor]]] = None
+    hidden_states: Optional[Tuple[torch.FloatTensor, ...]] = None
+    attentions: Optional[Tuple[torch.FloatTensor, ...]] = None
+    infer_request: Optional[openvino.runtime.InferRequest] = None
```

Comment:
could we rename it to something like `request` or `inference_request` ?

Reply:
current name is aligned with openvino api name, so for me infer_request sounds better

Reply:
I think that'd be clearer for users who are not familiar with the openvino ecosystem, also we don't use `infer_request` anywhere in optimum-intel so was thinking about something a bit more explicit